source/computer/macos-setup.rst

```diff
@@ -160,6 +160,28 @@ C
 Command Line Tools for Xcode 已经提供了 C 编译器和相关工具，因而无需单独安装
 C 编译器。
 
+.. note::
+
+    Command Line Tools for Xcode 提供的 C 编译器本质上是 Apple Clang 编译器，
+    其与 GCC 编译器有差异，但足以满足日常科研中编译 C 程序的需求。因而一般
+    用户无需再安装 GCC 编译器。
+
+    由于特殊原因需要安装 GCC 编译器的用户，可以使用如下命令安装::
+
+        $ brew install gcc
+
+    通过 Homebrew 安装的 GCC 提供了命令 ``gcc-10`` 和 ``g++-10``
+    （``10`` 是 GCC 的主版本号），以避免替换 Command Line Tools for Xcode
+    提供的 ``gcc`` 和 ``g++`` 命令。
+    为了使用 GCC 编译器，用户可以在 :file:`/usr/local/bin/` 目录下创建如下
+    软链接::
+
+        $ cd /usr/local/bin/
+        $ ln -s gcc-10 gcc
+        $ ln -s g++-10 g++
```

Comment:
Maybe we can just use `gcc-10` and `g++-10` in Makefile.

Reply:
Yes, but for simple codes, I belive Apple Clang is enough. For complicated codes, usually you can't find Makefiles.

Reply:
I mean we can use Clang. When we want to use GCC, we can directly use `gcc-10` and `g++-10` instead of setting symbol links.

Reply:
Done in 84aadf252c094d1aa61e34804d0b140da13bb1fc.